config/action.d/htaccess.conf

```diff
@@ -0,0 +1,66 @@
+# Fail2Ban configuration file
+#
```

Comment:
Please give a brief description for the action here and hints on what apache settings must be inplace to make it work